library/common/extensions/filters/http/platform_bridge/filter.cc

```diff
@@ -504,6 +514,18 @@ void PlatformBridgeFilter::resumeEncoding() {
   });
 }
 
+void PlatformBridgeFilter::resetIdleTimer() {
+  ENVOY_LOG(trace, "PlatformBridgeFilter({})::resetIdleTimer", filter_name_);
+
+  auto weak_self = weak_from_this();
+  dispatcher_.post([weak_self]() -> void {
+    if (auto self = weak_self.lock()) {
+      // Stream idle timeout is nondirectional.
+      // self->decoder_callbacks_->resetIdleTimer();
```

Comment:
Shouldn't this be uncommented?

Reply:
It's commented because it relies on @junr03's PR landing.